src/components/Lesson/Lesson.scss

```diff
@@ -0,0 +1,5 @@
+.container {
+  max-width: 90%;
```

Comment:
På sidene i dag har vi max-width på 800px. Synes vi skal fortsette med dette for nå.